ci-operator/config/openshift/openshift-docs/openshift-openshift-docs-enterprise-4.14.yaml

```diff
@@ -15,7 +15,7 @@ tests:
 - as: deploy-preview
   steps:
     env:
-      DISTROS: openshift-enterprise openshift-rosa openshift-dedicated microshift
+      DISTROS: openshift-enterprise microshift
       PREVIEW_COMMENT: site
```

Comment:
Need to update the 4.16 yaml with this field. Add it and run the builds again. Should be fine. I'll update the openshift-docs script this week.